src/home/buttons/detection-button.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Button, Grid } from '@material-ui/core'
+
+function cameras() {
+  window.location.href = '/detections'
+}
+
+export const DetectionButton = ({ children }) => <Grid item>
```

Comment:
Let's rename this component to `Detection`

Reply:
done